backbone/build.gradle

```diff
@@ -33,7 +33,7 @@ android {
         exclude 'META-INF/NOTICE.txt'
     }
 
-    resourcePrefix 'rsb_'
+//    resourcePrefix 'rsb_'
```

Comment:
Are you seeing warnings building just RS? Or in mPower with the submodule structure? If the latter, I have a fix locally I can add here